### PR TITLE
Handle multiple markers at the same location or close to each other on the map

### PR DESCRIPTION
**Pull request by MauPalantir**
Testing steps

Currently the feature is enabled for all views by default.

1. Without grouping:
Open the profiles view
Set the zoom level to a distant one so that there are overlapping markers.
Click on a marker which is very near to another. 
You should see a popup showing nearby locations.

2. Test grouped list:
Add the Features view to the map.
Find a place which has a linguistic feature and profile as well (eg. Tunis).
Click on the marker.
You should see a popup with the data for the nearby locations grouped by Location name and data type (profile, feature etc.)

3. Default behavior
Click on a marker which doesn't have anything nearby.
The marker should opne the content right away. 

4. Zoom
Try zooming into a cluster until the markers are distant.
The marlers should not open the popup list anymore.

### CODE REVIEW

File: components/geo-map.client.vue [hunk 1]
```diff
@@ -37,7 +61,62 @@ const context: GeoMapContext = {
...
+		componentPopups.value.push({
+			id: id as string,
+			props: {
+				markers: markers,
+				groupMarkers: contentTypes.length > 1,
+			},
+		});
+	} else if (marker.getPopup()) {
+		console.log(marker);
+		marker.removePopup();
```
Comment (ctot-nondef) [inline components/geo-map.client.vue:114]: I couldn't find the `removePopup()` method in the leaflet documentation, but also wasn't able to trigger this condition

Comment (MauPalantir) [reply components/geo-map.client.vue:114]: right, that seems unnecessary as markers are redrawn. Good catch!

Comment (ctot-nondef) [reply components/geo-map.client.vue:114]: so, will you keep working on this branch or should I merge and delete it?

Comment (MauPalantir) [reply components/geo-map.client.vue:114]: Yes, I should just remove that.

**Pull request merged**